env

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+function prepend {
+    local var=$1; shift
+    local sep=$1; shift
+    local i; for i in "$@"; do
+        if test -z "${!var}"; then
+            export "$var=$i"
+        else
+            export "$var=$i$sep${!var}"
+        fi
+    done
+}
+
+thisdir=$(cd $(dirname $0) && pwd)
+
+prepend LUA_PATH ';' ${thisdir}/src/
+prepend PATH ':' ${thisdir}/tools
```

Comment:
nit: use "$thisdir/tools" -- the braces are unnecessary (though it's fine if you leave them in) but quoting ensures that it is seen as one argument, otherwise paths with spaces (i know right?) would break the script.

also as discussed i guess LUA_PATH needs /?.lua appended

lgtm though :), thank you!